### DISCUSSION TITLE
AI是否能取代新聞業?

### DISCUSSION THREAD

**Discussion by MingWei174** [1101-GS4719-C/Python-Community-GS4719-C#2057] [General]
https://crossing.cw.com.tw/article/18902
近期人工智慧技術快速起飛，使人類生活變得更加便利，卻也在市場中掀起一波AI取代潮。

我心中常常想著:「那有了 AI，我們還需要記者嗎？」

我認為倘若未來 AI 真能產生出一段文意通順的報導內容，我比較追求感官刺激，所以AI生成的「單純陳述事實」的新聞內容對於我來說是稍顯不足的，但我也無法接受新聞媒體利用聳動、吸睛的「釣魚標題」騙取點擊率

在各種新聞播報題材中，我更喜歡記者在報導的同時穿插記者在目睹一件事當下的「個人」見解以及各種真實的反應，又或者是分享自己回憶種發生過的故事、場景描述的專題報導，透過故事的鋪陳，能讓我更快地進入報導情境，並且增加報導的可讀性。

我覺得「記者」的價值正是如此，一篇新聞報導之所以精彩，是因為字裡行間有著記者本人透過實際造訪、親自採訪下累積的個人見解。我始終相信，文字終究只是撰寫者與讀者的媒介，打動人心的往往是隱藏在文字背後的訊息。

因此，對於AI會完全取代記者，甚至是整個新聞業?我自己是否認這樣的說法，我認為記者能帶給我們比AI更有趣且更「人性化」的報導。

**Comment by hunteryang0108** [comment]
我認為以單純報導一個事件來看，AI在描述一個事件的能力確實是有可能勝過記者的。但是現在的AI還無法做到獨立去查證一個事件，更沒有辦法深入挖掘、訪談個案，這些都是AI無法取代新聞業的地方。

**Comment by peresonacifer** [comment]
我認為AI要完全取代新聞業還不可能的，必竟新聞報導由真人來講會較有溫度，甚至有些觀眾也會好奇主持人的反應，但要AI來處理文書方面或是編輯的工作應該還是綽綽有餘，事實的評論分析能力也是很頂的，因此可能導致新聞業有些工作的職缺會減少，這是難以避免的。

**Comment by Jerryguo888** [comment]
我認為AI難以完全取代記者，因為記者的價值在於他們能夠提供個人見解、實地觀察和生動的故事描述，這些是AI難以模仿的。相比之下，記者的專題報導和情境鋪陳能力比起AI能更好地打動讀者心靈。
